Hardcode version to avoid using git. Also don't do 'git clean'.

Index: bees-0.7/Makefile
===================================================================
--- bees-0.7.orig/Makefile
+++ bees-0.7/Makefile
@@ -7,7 +7,7 @@ LIBEXEC_PREFIX ?= $(LIB_PREFIX)/bees
 
 SYSTEMD_SYSTEM_UNIT_DIR ?= $(shell pkg-config systemd --variable=systemdsystemunitdir)
 
-BEES_VERSION ?= $(shell git describe --always --dirty || echo UNKNOWN)
+BEES_VERSION = 0.7
 
 # allow local configuration to override above variables
 -include localconf
